table.rst

```diff
@@ -0,0 +1,43 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              |  Int8Dtype,Int16Dtype,  |
+|                   |  Int32Dtype,Int54Dtype, |
++-------------------+-------------------------+
+|  str              | StringDtype,str         |
++-------------------+-------------------------+
+|  bytes            | BolleanDtype            |
++-------------------+-------------------------+
+|  float            |Float32Dtype,Float64Dtype|
++-------------------+-------------------------+
+|  complex          |                         |
++-------------------+-------------------------+
+|  Timestamp        |  Timestamp              |
```

Comment:
this can't be an argument to `astype()`, and I think that if we remove `Timestamp` from `S1`, all of our tests will pass, because we have `TimestampSeries`.  Need to test that.